lms/views/lti/basic_launch.py

```diff
@@ -248,8 +248,11 @@ def _show_document(self, assignment):
         # If there are any Hypothesis client feature flags that need to be
         # enabled based on the current application instance settings, those
         # should be enabled here via `self.context.js_config.enable_client_feature`.
-        #
-        # There are currently no such features.
+        ai_settings = self.request.lti_user.application_instance.settings
+        if ai_settings.get_setting(
+            ai_settings.fields[ai_settings.Settings.HYPOTHESIS_MENTIONS]
+        ):
+            self.context.js_config.enable_client_feature("mentions")
```

Comment:
@acelaya This is just a draft for now, but is this the right name for the client?

Reply:
We called it `at_mentions` in `h`.

![image](https://github.com/user-attachments/assets/241c09bd-2868-4ce4-b861-788e3401df7b)